manifest.php

```diff
@@ -25,10 +25,10 @@
     'label' => 'Linear Test Model',
     'description' => 'A simple linear test model',
     'license' => 'GPL-2.0',
-    'version' => '3.0.0',
+    'version' => '3.1.0',
     'author' => 'Open Assessment Technologies SA',
     'requires' => array(
-        'tao' => '>=9.0.0',
+        'tao' => '>=14.3.1',
```

Comment:
why 14.3.1? new feature was created in 12.21.0

Reply:
I checked some when was merged and it was 14.3.1 but I was wrong it's actually 12.21.4.
https://github.com/oat-sa/tao-core/pull/1477